WcaOnRails/app/models/competition.rb

```diff
@@ -48,6 +51,7 @@ class Competition < ActiveRecord::Base
     cellName
     showAtAll
     isConfirmed
+    eventSpecs
```

Comment:
Awesome, thanks! I should have been more clear: could you add a ruby comment to this line of code? That way, should someone stumble across this in the mean time, anyone reading this file won't be too confused.